R/model_parameters_mixed.R

```diff
@@ -9,6 +9,12 @@
 #' @param wb_component Logical, if \code{TRUE} and models contains within- and between-effects (see \code{\link{demean}}), the \code{Component} column will indicate which variables belong to the within-effects, between-effects, and cross-level interactions. By default, the \code{Component} column indicates, which parameters belong to the conditional or zero-inflated component of the model.
 #' @inheritParams model_parameters.default
 #'
+#' @details
+#' For mixed-effect \code{afev_aov} modelss, the underlying \code{aov} object is
+#' used (which required the model be fit with \code{include_aov = TRUE}), which
+#' might give slightly different results compared the anova table provided by
+#' \code{afex}.
+#'
```

Comment:
does this apply to mixed models, i.e. to `afex::mixed()`? I thought this only applies to anova-models from afex... So shouldn't this move to the docs from `model_parameters.aov()`?

Reply:
No, this only applies to `afex::aov_car()` models with within-subject effects, i.e., mixed effects ANOVAs.

Reply:
I mean, the need to `include_aov = TRUE` is true for any `afex::aov_car()` model,
but the fact that results may differ only applies to `afex::aov_car()` models with within-subject effects...

easy! :P

Reply:
But still it produces ANOVA tables and is processed in the model_parameters.aov, so I would put it there in the docs. The model_parameters.mixed is only for afex::mixed()

Reply:
Ohhh.... okay, I'll fix it up and move it.